vm/resolutions.go

```diff
@@ -175,38 +175,6 @@ func (vm *VM) processAcceptedBlock(b *chain.StatelessBlock) {
 		vm.Fatal("accepted processing failed", zap.Error(err))
 	}
 
-	// Sign and store any warp messages (regardless if validator now, may become one)
-	results := b.Results()
-	for i, tx := range b.Txs {
-		// Only cache auth for accepted blocks to prevent cache manipulation from RPC submissions
-		vm.cacheAuth(tx.Auth)
```

Comment:
This should still be done. This is unrelated to warp.

Reply:
That being said, this was a remnant of earlier crypto libs we used and it may be better to just remove this functionality from the auth interface.

In either case, we should do one or the other.

Reply:
I kept the `vm.cacheAuth`